### PR TITLE
remove nelmio api-doc from composer

### PR DESCRIPTION
**Pull request by jmleroux**
This PR finalize #4360 because i forgot to remove the bundle from composer.

### CODE REVIEW

**Comment by nidup** [comment]
GTM

File: app/AppKernel.php [hunk 1]
```diff
@@ -158,7 +158,6 @@ protected function getOroDependencies()
             new JMS\SerializerBundle\JMSSerializerBundle(),
             new Knp\Bundle\MenuBundle\KnpMenuBundle(),
             new Liip\ImagineBundle\LiipImagineBundle(),
-            new Nelmio\ApiDocBundle\NelmioApiDocBundle(),
```
Comment (jmleroux) [inline app/AppKernel.php]: I didn't notice this one ! :scream: 
I hope it'll not ruin the previous work.

**Comment by jmleroux** [comment]
> GTM

@nidup , not so sure. :sweat: 
There was a dependency i didn't see...

**Comment by nidup** [comment]
Please add this remove of dep in the upgrade-1.6.md

**Comment by jmleroux** [comment]
> Please add this remove of dep in the upgrade-1.6.md

Oh yes, good catch !

**Comment by jmleroux** [comment]
@nidup , i added it in the EE upgrade.md, but i think it's not necessary here because the composer.json is overwriten withe new one [here](https://github.com/akeneo/pim-community-dev/blob/master/UPGRADE-1.6.md#update-dependencies-and-configuration)

**Comment by jjanvier** [comment]
gtm

**Pull request merged**